src/simple.jl

```diff
@@ -21,7 +21,7 @@ end
 
 @inline converged(p::PowerMethodIterable) = p.residual ≤ p.tol
 
-@inline start(p::PowerMethodIterable) = 0
+@inline start(p::PowerMethodIterable) = 1
```

Comment:
Alternatively,  we can do `reserve!(history, :resnorm, maxiter + 1)` but iterating `maxiter + 1` instead of `maxiter` was a minor bug so why not fix it here.